From 3defaae29edd09e89ea0d05082a7a8d5d2cb48d0 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Sat, 10 Sep 2016 04:26:24 +0300
Subject: [PATCH 02/15] configs: rcar-gen3-common: Enable U-Boot scripts

This enables "source" command, which can be used
for running U-Boot scripts in order to simplify
flashing procedure.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 include/configs/rcar-gen3-common.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/rcar-gen3-common.h b/include/configs/rcar-gen3-common.h
index 50cfa8a..bab2c2c 100644
--- a/include/configs/rcar-gen3-common.h
+++ b/include/configs/rcar-gen3-common.h
@@ -33,6 +33,8 @@
 #define CONFIG_CMD_EXT4_WRITE
 #define CONFIG_CMD_GPIO
 
+#define CONFIG_CMD_SOURCE
+
 #define CONFIG_SYS_THUMB_BUILD
 #define CONFIG_SYS_GENERIC_BOARD
 
-- 
1.9.3

